python/vyos/ipsec.py

```diff
@@ -83,6 +83,35 @@ def get_vici_sas_by_name(ike_name: str, tunnel: str) -> list:
         raise ViciCommandError(f'Failed to get SAs')
 
 
+def get_vici_connection_by_name(ike_name: str) -> list:
+    """
+    Find sas by IKE_SA name and/or CHILD_SA name
+    and return list of OrdinaryDicts with SASs info
+    If tunnel is not None return value is list of OrdenaryDicts contained only
```

Comment:
Do we need to re-tell this if type tags already contain that information? It might be better to write types in the Python 3.10's typing notation, too.

I don't consider it a blocker for merging, just a style thing.